q2cli/cli.py

```diff
@@ -0,0 +1,104 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+import qiime
+from qiime.sdk import PluginManager, SubprocessExecutor
+
+from . import __version__ as q2cli_version
+
+@click.group()
+@click.pass_context
+def cli(ctx):
+    ctx.obj['plugin_manager'] = PluginManager()
```

Comment:
I can't seem to get this working right now - I think the group's context isn't active when I call ``build_command``, which might mean that it would make more sense for that to happen as part of ``cli`` itself. I'm just going to leave this as-is for the moment, since it doesn't impact any UI stuff.